internal/rbd/rbd_util.go

```diff
@@ -288,12 +287,17 @@ func deleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 	}
 
 	if !rbdCephMgrSupported {
-		// attempt older style deletion
-		args := []string{"rm", image, "--pool", pOpts.Pool, "--id", cr.ID, "-m", pOpts.Monitors,
-			"--keyfile=" + cr.KeyFile}
-		output, err = execCommand("rbd", args)
+		var ioctx *rados.IOContext
+		ioctx, err = pOpts.conn.GetIoctx(pOpts.Pool)
+		if err != nil {
+			return err
+		}
+		defer ioctx.Destroy()
+
+		image := librbd.GetImage(ioctx, pOpts.RbdImageName)
```

Comment:
we already have a variable called image here https://github.com/ceph/ceph-csi/blob/1f0f2c04e12bb931b8d634aace760d73d2a75a0d/internal/rbd/rbd_util.go#L270. rename this to something else

Reply:
Interesting, the linters are so strict, and mostly complain about these kind of things. Somehow this sneaked through!